crates/holochain/src/conductor/conductor.rs

```diff
@@ -721,10 +718,10 @@ mod dna_impls {
 
         /// Restart every paused app
         #[cfg_attr(feature = "instrument", tracing::instrument(skip_all))]
-        pub(crate) async fn start_paused_apps(&self) -> ConductorResult<AppStatusFx> {
+        pub(crate) async fn start_stopped_apps(&self) -> ConductorResult<AppStatusFx> {
```

Comment:
Has the last usage of this not been removed above?

Semantically, starting a paused app is quite different from pausing a stopped app. We aren't distinguishing between failed to start and deliberately stopped by the user. I think that's fine but then we shouldn't just start all stopped apps, we should let the user selectively start the apps they want?

I think this can be removed, if it can't, I think it might make sense to try to remove it.